torch/distributed/algorithms/ddp_comm_hooks/powerSGD_hook.py

```diff
@@ -97,9 +101,7 @@ def _should_compress(
 
 
 def _report_compression_stats(bucket, state):
-    """
-    Report compression stats at the frequency of `compression_stats_logging_frequency` specified in PowerSGD state.
-    """
+    """Report compression stats at frequency of `compression_stats_logging_frequency` specified in PowerSGD state."""
```

Comment:
Does this need double backticks (\`\`) around `compression_stats_logging_frequency`?

Reply:
Yes I believe so to maintain consistency with other documentation in file.